### PR TITLE
Create tables - migrations

### PR DESCRIPTION
**Pull request by nirzalcman**

### CODE REVIEW

File: src/main/resources/db/migration/V202409231010__create_table_feedback.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CREATE TABLE feedback
+(
+    id SERIAL PRIMARY KEY ,
+    company_id BIGINT NOT NULL,
+    employee_id BIGINT NOT NULL ,
```
Comment (rakreshet) [inline src/main/resources/db/migration/V202409231010__create_table_feedback.sql]: if it's annoymous, are we going to save the id? I'm not sure. I missed that in the HLD. So maybe we want to allow it to be Null

Comment (nirzalcman) [reply src/main/resources/db/migration/V202409231010__create_table_feedback.sql]: yes it should be null in this case i added migration for this now .

Comment (rakreshet) [reply src/main/resources/db/migration/V202409231010__create_table_feedback.sql]: but I see: NOT NULL :). maybe you forgot to remove it?

Comment (nirzalcman) [reply src/main/resources/db/migration/V202409231010__create_table_feedback.sql]: I DID ANOTHER MIGRATION FOR THIS TO DROP THE NOT NULL

File: src/main/resources/db/migration/V202409231115__drop_notnull_in_employee_id_col_in_feedback_table.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE feedback
+ALTER COLUMN employee_id DROP NOT NULL;
```
Comment (rakreshet) [inline src/main/resources/db/migration/V202409231115__drop_notnull_in_employee_id_col_in_feedback_table.sql]: redundant, just remove null from the original table creation

**Review by rakreshet — APPROVED** [review]

**Pull request merged**